nixos/modules/services/web-apps/birdsitelive.nix

```
@@ -0,0 +1,447 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.services.birdsitelive;
+  settingsType = (pkgs.formats.json { }).type;
+
+  isSecret = v: with builtins; isAttrs v && v ? _secret && isString v._secret;
+  secret = with lib; mkOptionType {
+    name = "secret";
+    description = "secret value";
+    descriptionClass = "noun";
+    check = isSecret;
+    getSubOptions = prefix: prefix ++ [ "_secret" ];
+    nestedTypes = {
+      _secret = types.str;
+    };
+  };
+
+  toJsonSec =
+    let toJsonSec' = {}@args: v: with builtins;
```

```suggestion
    let toJsonSec' = {}@args: v: with lib;
```
lib versions of functions are preferred over builtins.

---

Corrected.